src/pages/guidelines/ibm-dot-com-style-models/overview.mdx

```diff
@@ -0,0 +1,131 @@
+
+<Title>Primary user intent</Title>
+
+For any given page, there is a overarching user intent—we call this the primary user intent of a page. For an example, learn is the primary user intent for the early learn search capture page.
+
+
+<Title>Grid usage</Title>
+
+The grid system creates the fundamental structure of a page. There are many different ways to use the IBM 2X Grid, and the style models narrow down the available options to a few optimized.
+
+<Row>
+<Column colMd={12} colLg={12}>
+
+![Grid usage examples](./images/grid-usage.png)
+
+</Column>
+</Row>
+
+<Title>Primary color theme</Title>
+
+In IBM.com style models, the color theme is a major way to create the sense of progression over the user&lsquo;s journey. In general, the white theme is used for pre-purchase pages, the Gray 10 theme for the purchase and the post-purchase experience (My IBM). For the products, color themes are not specified in order to accomodate each product&lsquo;s needs the best as the user intents for the product experience can vary vastly. 
+
+<Row>
+<Column colMd={12} colLg={12}>
+
+![Reflecting journey progression](./images/color-themes.png)
+
+</Column>
+</Row>
+
+<Title>Text sizes and spacing</Title>
+
+Different user intent requires different type scale and spacing. For an example, for the early learn page, which requires heavy reading of long-form content, benefits from larger text sizes and more open spaces. On the other hand, the checkout experiences are etter served with smaller text sizes and tighter spaces.
+
+<Row>
+<Column colMd={8} colLg={8}>
+
+![Reflecting journey progression](./images/type-scales-spacing.png)
+
+</Column>
+</Row>
+
+<Title>Layout and screen regions</Title>
+
+Layout theme lays out the overall layout for any given user intent by defining the regions for navigational elements, main content, supporting content, and key visual anchors placements. By using these layouts, the the user will not have to learn how to use each page as they land on it.
+
+<Row>
+<Column colMd={8} colLg={8}>
+
+![Reflecting journey progression](./images/screen-region-examples.png)
+
+</Column>
+</Row>
+
+## Using the style models
+
+There are currently 9 user intents identified so far, each with an associated style model. As a new page is designed, this should be the starting point.
+
+1. Find the matching user intent of the page you are designing from the [list](./style-model-list#user-intents-and-style-models).
+2. Use the associated style model to design your page.
+
+<Title>Feedback</Title>
+
+These user intents and style models are not locked in stone—these are intended to evolve over time. If for any reason, the existing user intents or style models do not meet the needs, please contact the Digital Design System team via the #ibm-digital-design slack channel.
```

Comment:
We might need @JennySanchez to help us link to the slack channel here! I don't know how 😅

Reply:
A good catch. Link committed.